bindings/nodejs/lib/types/wallet/bridge/index.ts

```diff
@@ -2,7 +2,6 @@ import type {
     __AnnounceCandidacyMethod__,
     __PrepareBurnMethod__,
     __PrepareClaimOutputsMethod__,
-    __ClaimOutputsMethod__,
```

Comment:
And the actual core binding would be removed later?

Reply:
Added it here https://github.com/iotaledger/iota-sdk/issues/2081